fragment_downloaded_cb({"url": "c/testing.html#g_test_failed1", "fragment": "g_test_failed \nReturns whether a test has already failed. This will be the case when g_test_fail g_test_incomplete or g_test_skip have been called but also if an assertion has failed. \nThis can be useful to return early from a test if continuing after a failed assertion might be harmful. \nThe return value of this function is only meaningful if it is called from inside a test function. \n"});